merge_requests.go

```diff
@@ -169,6 +169,8 @@ type ListMergeRequestsOptions struct {
 	SourceBranch    *string    `url:"source_branch,omitempty" json:"source_branch,omitempty"`
 	TargetBranch    *string    `url:"target_branch,omitempty" json:"target_branch,omitempty"`
 	Search          *string    `url:"search,omitempty" json:"search,omitempty"`
+	In              *string    `url:"in,omitempty" json:"in,omitempty"`
+	Wip             *string    `url:"wip,omitempty" json:"wip,omitempty"`
```

Comment:
In Go [initialisms and acronyms are written with all caps](https://github.com/golang/go/wiki/CodeReviewComments#initialisms), so this should be `WIP`

Reply:
Just pushed an update. Thanks for the link.